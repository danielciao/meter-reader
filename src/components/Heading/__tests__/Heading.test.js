import React from 'react';
import renderer from 'react-test-renderer';

import { H1, H2 } from '../index';

describe('Heading', () => {
  it('should render h1 correctly', () => {
    const result = renderer.create(<H1>This is a heading</H1>).toJSON();

    expect(result).toMatchSnapshot();
  });

  it('should render h2 correctly', () => {
    const result = renderer.create(<H2>This is a heading</H2>).toJSON();

    expect(result).toMatchSnapshot();
  });
});
