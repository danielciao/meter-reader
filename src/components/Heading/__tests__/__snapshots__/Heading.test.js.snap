// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should render h1 correctly 1`] = `
<h1
  className="sc-bdVaJa Egqup"
>
  This is a heading
</h1>
`;

exports[`Heading should render h2 correctly 1`] = `
<h2
  className="sc-bwzfXH lhwasE"
>
  This is a heading
</h2>
`;
